pandas/tests/computation/test_eval.py

```diff
@@ -774,16 +774,19 @@ def check_chained_cmp_op(self, lhs, cmp1, mid, cmp2, rhs):
 f = lambda *args, **kwargs: np.random.randn()
 
 
-ENGINES_PARSERS = list(product(_engines, expr._parsers))
-
```

Comment:
I've made the change, but I'll wait until the network test is corrected before updating.